src/MLJGLMInterface.jl

```diff
@@ -17,7 +17,7 @@ export LinearRegressor, LinearBinaryClassifier, LinearCountRegressor
 import MLJModelInterface
 import MLJModelInterface: metadata_pkg, metadata_model, Table, Continuous, Count, Finite,
     OrderedFactor, Multiclass, @mlj_model
-import Distributions
+import Distributions: Bernoulli, Distribution, Poisson
```

Comment:
```suggestion
using Distributions: Bernoulli, Distribution, Poisson
```
When not overriding or extending methods, let's go for `using` for clarity